README.md

```diff
@@ -24,3 +24,22 @@ user.save!
 6. Fill the rest of the form and submit it.
 
 You're good to go!
+
+## Testing
+
+Run `rake decidim:generate_external_test_app` to generate a dummy application to test your module.
+
+Require missing factories in `./spec/factories.rb`
+
+Add `require "rails_helper"` to your specs and execute them from the root directory, i.e.:
+
+`rspec ./decidim-process-extended/spec/serializers/decidim/process/extended/process_serializer_spec.rb`
+
+## Open Data
+
+Public information from Processes and Normative consultations can be accessed at [placeholder](https://github.com/gencat/participa/tree/feat/socrata_metadata_exporter#open-data), which is powered by [Socrata](https://socrata.com/).
+
+There are two rake tasks available to interact with this data:
+
+- `rake socrata:export` to export the data to a file
```

Comment:
As a namespace, I like `export` less than `socrata`.

How about this?
- `participa:export_socrata_dataset`
- `participa:update_socrata_dataset`

Reply:
I think that exporting to a file is not related with socrata. Integration with Socrata may be removed but it still has sense to have this export rake task.
What about..?:
- `export:participatory_processes`
- `socrata:publish:participatory_processes`

Reply:
Forgive me to enter to this discussion, but I will give my opinion :)

I understand that the two rake tasks are part of the open data namespace, no?

One of them, exports the processes of "Participa" to a CSV, right?
The other one, publish to Socrata the processes of "Participa"? 

Keep in mind these premises, What about?

- `open_data:export:participatory_processess`
- `open_data:publish_to_socrata:participatory_processes`

So, if, in a future is necessary to extended the functionality to assemblies, it would be:
- `open_data:export:assemblies`
- `open_data:publish_to_socrata:assemblies`

Reply:
I agree with @isaacmg410 , what about you @tramuntanal ?

Reply:
Great! I also like @isaacmg410 's proposal :D